osbs/api.py

```diff
@@ -872,10 +872,19 @@ def _load_quota_json(self, quota_name=None):
         return quota_json['metadata']['name'], quota_json
 
     @osbsapi
-    def pause_builds(self, quota_name=None):
+    def pause_builds(self, quota_name=None, ignore_quota_errors=True):
         # First, set quota so 0 pods are allowed to be running
         quota_name, quota_json = self._load_quota_json(quota_name)
-        self.os.create_resource_quota(quota_name, quota_json)
+        try:
+            self.os.create_resource_quota(quota_name, quota_json)
+        except OsbsResponseException as e:
+            if e.status_code == 403:
```

Comment:
Done, now its `if ignore_quota_errors and (e.status_code == requests.codes.FORBIDDEN):` to avoid magic numbers too